samples/XMasControlAddIn/Pages/XmasGreetingPart.al

```diff
@@ -0,0 +1,33 @@
+page 50101 "X-mas Greeting Part"

```

Comment:
I would prefer to call it XmasGreetingPart and use the nicer name in the caption. This fits better to the new language model

````VisualBasic
page 50101 XmasGreetingPart
````